packages/conf-python-3/conf-python-3.9.0.0/opam

```diff
@@ -17,7 +17,7 @@ depexts: [
   ["dev-lang/python:3.6"] {os-distribution = "gentoo"}
   ["python3"] {os = "openbsd"}
   ["lang/python39"] {os = "netbsd"}
-  ["lang/python39"] {os = "freebsd"}
+  ["python39"] {os = "freebsd"}
```

Comment:
```suggestion
  ["lang/python39"] {os = "freebsd"} # this will only result in a python3.9 command and would not give python3 command
```
Revert changes